### PR TITLE
add SayStyle() and SayOption()

### PR DESCRIPTION
**Pull request by fdarosa2663**
Adding "style" and "option" with number prompts to menu to help users identify and select the style and options easier.

### CODE REVIEW

File: props/saber_fett263_buttons.h [hunk 1]
```diff
@@ -2617,14 +2627,18 @@ SaberFett263Buttons() : PropBase() {}
...
       case IGNITION_OPTION_ARG:
 	menu_type_ = MENU_IGNITION_OPTION;
 	arg_revert_ = strtol (argspace, NULL, 0);
+        sound_library_.SayOption();
+        sound_library_.SayNumber(calc_, SAY_WHOLE);
 	break;
       case IGNITION_TIME_ARG:
 	menu_type_ = MENU_IGNITION_TIME;
-	arg_revert_ = strtol (argspace, NULL, 0);
+	arg_revert_ = strtol (argspace, NULL, 0);
```
Comment (profezzorn) [inline props/saber_fett263_buttons.h:2641]: Looks like you just added some whitespace at the end of the line here. Please fix.

File: props/saber_fett263_buttons.h [hunk 2]
```diff
@@ -2852,8 +2870,16 @@ SaberFett263Buttons() : PropBase() {}
         itoa(style_num_, style_arg, 10);
         current_preset_.SetStyle(blade_num_, style_parser.SetArgument(current_preset_.GetStyle(blade_num_), 1, style_arg));
         current_preset_.Save();
-        sound_library_.SayNumber(style_num_, SAY_WHOLE);
         UpdateStyle(current_preset_.preset_num);
+        sound_library_.SayStyle();
+        sound_library_.SayNumber(style_num_, SAY_WHOLE);
```
Comment (profezzorn) [inline props/saber_fett263_buttons.h:2875]: This part looks very similar to line 2119 above, maybe break it out into a helper function and save a few bytes?

File: props/saber_fett263_buttons.h (hunk 1 above)
Comment (fdarosa2663) [reply props/saber_fett263_buttons.h]: fixed

File: props/saber_fett263_buttons.h (hunk 2 above)
Comment (fdarosa2663) [reply props/saber_fett263_buttons.h]: broke out to SayStyleNumber()

**Pull request merged**